runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java

```diff
@@ -82,10 +83,21 @@
 	private final RuneLiteProperties properties;
 	private JPanel navContainer;
 	private PluginToolbar pluginToolbar;
+	private JPanel container;
+
+	@Getter
 	private PluginPanel pluginPanel;
 
+	@Getter
+	@Setter
+	private PluginPanel lastPluginPanel;
```

Comment:
Can this be done in better way without creating web of dependencies between everything like this?